content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: "Using the grype Container Scanner"
+type: "article"
+linktitle: "grype-tutorial"
+aliases: 
+- /chainguard/chainguard-images/working-with-images/scanners/grype-tutorial
+description: "Learn to use grype to detect CVEs in images"
+date: 2024-06-06:08:00+02:00
+lastmod: 2024-06-06:08:00+00:00
+tags: ["Conceptual", "CVE"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "scanners"
+weight: 100
+toc: true
+---
+
+[grype](https://github.com/anchore/grype) is a vulnerability scanner for container images and filesystems developed and maintained by [Anchore](https://anchore.com/) and written in the Go programming language. grype can scan from Docker, OCI, Singularity, podman, image archives, and local directory. grype is compatible with SBOMs generated by [syft](https://github.com/anchore/syft), and grype's [vulnerability database](https://github.com/anchore/grype-db) draws from a wide variety of sources, including [Wolfi SecDB]( https://github.com/wolfi-dev/wolfictl/blob/main/docs/cmd/wolfictl_advisory_secdb.md).
+
+grype is appropriate for one-off detection for manual CVE mitigation and in automated use in CI pipelines. Chainguard maintains a [low-to-no CVE Chainguard Image for grype](https://edu.chainguard.dev/chainguard/chainguard-images/reference/grype/) based on our lightweight Wolfi distribution.
```

Comment:
```suggestion
grype is appropriate for one-off detection for manual CVE mitigation and in automated use in CI pipelines. Chainguard maintains a [low-to-no CVE Chainguard Image for grype](/chainguard/chainguard-images/reference/grype/) based on our lightweight Wolfi distribution.
```
don't forget - we always use relative links for Academy content

Reply:
Good catch.